src/EncompassRest/Loans/LoanApiObject.cs

```diff
@@ -78,7 +78,7 @@ internal LoanApiObject(EncompassRestClient client, string loanId, string baseApi
         internal async Task UpdateAsync<TClass>(TClass value, string methodName, bool populate, CancellationToken cancellationToken)
             where TClass : class, T, IDirty, IIdentifiable
         {
-            await PatchPopulateDirtyAsync(value.Id, JsonStreamContent.Create(value), methodName, value.Id, cancellationToken, value, populate).ConfigureAwait(false);
+            await PatchPopulateDirtyAsync(value.Id, populate ? ViewEntityQueryString : null, JsonStreamContent.Create(value), methodName, value.Id, cancellationToken, value, populate).ConfigureAwait(false);
```

Comment:
we need to allow for more querystring options somehow.

Reply:
Agreed

Reply:
Actually, I don't think that's necessary and would just make these API's harder to implement. I think if we ever come across an instance in the future where we need to allow for more querystring options we can just add overloads.

Reply:
ok.  i was just thinking about future proofing.